gce/survival-training/README.md

```diff
@@ -0,0 +1,113 @@
+# Compute Engine survival training
+
+Training jobs that survive Compute Engine shutdowns
```

Comment:
Google Compute Engine? And in the title as well?

Reply:
As per @thedriftofwords , it should just be "Compute Engine".

Reply:
It could either be "Google Compute Engine" or "Compute Engine", just not "GCE"